config_automation.yml

```diff
@@ -16,8 +16,8 @@ docker-test: no
 ##### Renderings run upon merge to main branch #####
 # Rendering each platform's content
 render-bookdown: yes
-render-leanpub: no
-render-coursera: no
+render-leanpub: yes
```

Comment:
I'll set these back to the default of `no` before merging. This was for testing purposes.